### PR TITLE
Fixed small Borzoi discrepancies, fixes #144

### LINKED ISSUE #144: borzoi model
Hi,

First of all thank you for your efforts putting together such a great framework for DNA sequence deep learning models. I have a simple question about the pytorch version of borzoi model. Did you remove all the dropout layers in the transformer blocks?

Based on the supplementary figure 14 in the borzoi paper, they use dropout with p = 0.2 in the transformer blocks. But when I examine the model structure from grelu, the dropout layers are all identity functions.

Thank you for your help.

Image

```
      (transformer_tower): TransformerTower(
        (blocks): ModuleList(
          (0-7): 8 x TransformerBlock(
            (norm): Norm(
              (layer): LayerNorm((1536,), eps=1e-05, elementwise_affine=True)
            )
            (mha): Attention(
              (to_q): Linear(in_features=1536, out_features=512, bias=False)
              (to_k): Linear(in_features=1536, out_features=512, bias=False)
              (to_v): Linear(in_features=1536, out_features=1536, bias=False)
              (to_out): Linear(in_features=1536, out_features=1536, bias=True)
              (to_pos_k): Linear(in_features=32, out_features=512, bias=False)
              (pos_dropout): Dropout(p=0.0, inplace=False)
              (attn_dropout): Dropout(p=0.0, inplace=False)
            )
            (dropout): Dropout(
              (layer): Identity()
            )
            (ffn): FeedForwardBlock(
              (dense1): LinearBlock(
                (norm): Norm(
                  (layer): LayerNorm((1536,), eps=1e-05, elementwise_affine=True)
                )
                (linear): Linear(in_features=1536, out_features=3072, bias=True)
                (dropout): Dropout(
                  (layer): Identity()
                )
                (act): Activation(
                  (layer): ReLU()
                )
              )
              (dense2): LinearBlock(
                (norm): Norm(
                  (layer): Identity()
                )
                (linear): Linear(in_features=3072, out_features=1536, bias=True)
                (dropout): Dropout(
                  (layer): Identity()
                )
                (act): Activation(
                  (layer): Identity()
                )
              )
            )
          )
        )
      )
```

### PR DESCRIPTION
**Pull request by johahi**
* Fixes norm epsilons of pretrained Borzoi models
* Fixes GELU approximation
* Fixes dropout config

### CODE REVIEW

File: src/grelu/model/trunks/borzoi.py [hunk 1]
```diff
@@ -201,7 +204,7 @@ def __init__(
             device=device,
             dtype=dtype,
         )
-        self.act = Activation("gelu")
+        self.act = Activation("gelu_borzoi")
```
[the remark this answers is not in the document]
Comment (avantikalal) [reply src/grelu/model/trunks/borzoi.py:207]: There's no act_func argument to this class. You can create a new PR to add one.
